README.md

```diff
@@ -16,21 +16,21 @@ Create a `.env` file with the content of the `.env.sample` and edit with your pe
 
 Navigate into the root directory of the project and run the script `setup.sh` with the command:
 ```
-./setup.sh
+./bash_scripts/setup.sh
```

Comment:
Didn't the acceptance criteria on Pivotal Tracker state that the folder should be called `scripts` I personally prefer the name scripts though as this is common to codebase I have worked on in the past and for the random projects I have worked on.

Reply:
While `bash_script` may look descriptive, you would still be able to relate with the file when you see a folder called script and what comes to mind will be bash script because all other files are scripts by default. I wouldn't want to go around naming our root folder `python_script` or `javascript_script` for consistency though, although that's just been paranoid let's not go through that route. just keep it simple. I understand your passion here. `scripts` should suffice and let's stick to the acceptance criteria.

Reply:
@andela-sjames we shall make the necessary changes to meet the acceptance criteria.

Reply:
For context, I did not come up with the final acceptance criteria, but I actually like the acceptance criteria.